app/bundles/LeadBundle/Entity/CompanyLeadRepository.php

```diff
@@ -45,12 +45,15 @@ public function saveEntities($entities, $new = true)
     /**
      * Get companies by leadId.
      *
-     * @param $leadId
-     * @param $companyId
+     * @param int       $leadId
+     * @param int|null  $companyId
+     * @param bool|null $onlyPrimary
      *
      * @return array
+     *
+     * @throws \Doctrine\DBAL\Exception
      */
-    public function getCompaniesByLeadId($leadId, $companyId = null)
+    public function getCompaniesByLeadId($leadId, $companyId = null, $onlyPrimary = null)
```

Comment:
Why not use false as the default value?
```suggestion
    public function getCompaniesByLeadId($leadId, $companyId = null, bool $onlyPrimary = false)
```